docs/book/src/introduction/core_library.md

```diff
@@ -0,0 +1,48 @@
+# Core Library
+
+The Sway Core Library, like the name suggests contains core operators and logic for the primitive types of the Sway programming language. These traits and methods are an extension of the [primitive types](https://docs.fuel.network/docs/sway/basics/built_in_types/#primitive-types) `u8`, `u16`, `u32`, `u64`, `u256`, `str[]`, `str`, `bool` and , `b256` and can be used where appropriate.
+
+The latest core library documentation can be found in the [Core Library Book](https://fuellabs.github.io/sway/master/core/). If the latest version is not compatible please refer to the appropriate tagged release.
```

Comment:
It's not the "Core Library Book". It's the core library documentation. Book implies that it's some handwritten markdown when it's autogenerated.
In fact we should probably mention this isn't special and was generated with `forc doc` the same way users' own docs can be.